src/registrar/models/domain.py

```diff
@@ -279,55 +282,173 @@ def _create_host(self, host, addrs):
             logger.error("Error _create_host, code was %s error was %s" % (e.code, e))
             return e.code
 
+    def _convert_list_to_dict(self, listToConvert: list[tuple[str]]):
+        newDict = {}
+
+        # TODO-848: If duplicated nameserver names, throw error
+        for tup in listToConvert:
+            if len(tup) == 1:
+                newDict[tup[0]] = None
+            else:
+                newDict[tup[0]] = tup[1]
+        return newDict
+
+    def isSubdomain(self, nameserver):
+        return nameserver.find(self.name) != -1
```

Comment:
**comment (non-blocking)**: I think a more pythonic version of this is `self.name not in nameserver`.

Reply:
Oop 30 minutes and 4 print statements later I just realized you meant `self.name in nameserver` not `not in` 😆 😭